test/taylorize.jl

```diff
@@ -5,6 +5,7 @@ using Test
 using LinearAlgebra: norm
 using InteractiveUtils: methodswith
 using Elliptic
+using Base.Threads
```

Comment:
We may need to add Base.Threads to the `[deps]` section, as well as to `[extras]` and `[targets]` in `Project.toml`.

Reply:
Thanks, I added Threads in the corresponding sections of Project.toml